src/index.html

```diff
@@ -6,8 +6,40 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./stars.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/rectangle-four.png"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__goods"
+      >
+
+      <p class="card__info">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card-chosen">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card-reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__buy">
+        <p class="card__price">Price:</p>
+        <p class="card__sum">$2,199</p>
+      </div>
+
+      <a href="#buy" data-qa="hover"><button class="card__button">buy</button></a>
```

Comment:
You can use only one tag `a` here
```suggestion
      <a href="#buy" data-qa="hover" class="card__button">
        buy
      </a>
```